handlers.go

```diff
@@ -250,7 +250,13 @@ func buildCommonLogLine(req *http.Request, url url.URL, ts time.Time, status int
 		host = req.RemoteAddr
 	}
 
-	uri := url.RequestURI()
+	uri := req.RequestURI
+	if req.ProtoMajor == 2 && req.Method == "CONNECT" {
```

Comment:
Add a comment here that references https://httpwg.github.io/specs/rfc7540.html#CONNECT - e.g.
```
// Requests using the CONNECT method over HTTP/2.0 must use the authority field (aka r.Host) to identify the target
```